ktor-client/ktor-client-core/common/src/io/ktor/client/response/HttpResponse.kt

```diff
@@ -76,6 +77,6 @@ interface HttpResponse : HttpMessage, CoroutineScope, Closeable {
  */
 suspend fun HttpResponse.readText(charset: Charset? = null): String {
     val packet = receive<ByteReadPacket>()
-    val actualCharset = charset() ?: charset ?: call.responseConfig.defaultCharset
+    val actualCharset = charset() ?: charset ?: call.client.feature(HttpPlainText)!!.responseCharsetFallback
```

Comment:
If no `HttpPlainText` feature installed this will fail. I'd use ` ?.responseCharsetFallback ?: Charsets.UTF_8)`